packages/connections/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@dcos/connections",
```

Comment:
Do we want to reflect the org structure in the folder structure?
```
packages
- @dcos
-- connections
```

Reply:
I don't think we need to reflect the org structure as all packages belong to `@dcos`

Reply:
Not all packages :) i.e. `recordio` and `mesos-client` aren't `@dcos` specific and could live freely. The thing is we either don't use `@dcos` prefix, or put it in a subdirectory as npm would put it in `node_modules`

Reply:
I'd simply "scope" all packages as they all belong to `dcos` including `recordio` and `mesos-client`.

Reply:
I'm going to remove the scope from the package for now to unblock the PR and will add the topic to our sync agenda.